src/RepairManagerAgent/logging.yaml

```diff
@@ -0,0 +1,28 @@
+---
+version: 1
+disable_existing_loggers: False
+formatters:
+  simple:
+    format: '%(asctime)s - %(levelname)s - %(filename)s:%(lineno)d - %(message)s'
+handlers:
+  console:
+    class: logging.StreamHandler
+    level: DEBUG
+    formatter: simple
+    stream: ext://sys.stdout
+  file:
+    class : logging.handlers.RotatingFileHandler
+    formatter: simple
+    filename: repairmanageragent.log
```

Comment:
`/var/log/dlworkspace/repairmanageragent.log`? I'm not sure where this logging file will be when you use relative path.

Reply:
it gets logged inside the docker container... oops